fedimint-ui/apps/guardian-ui/src/components/RunDKG.tsx

```diff
@@ -84,7 +84,7 @@ export const RunDKG: React.FC<Props> = ({ next }) => {
       <CircularProgress
         isIndeterminate={!isWaitingForOthers}
         value={isWaitingForOthers ? progress : undefined}
-        color={theme.colors.blue[400]}
+        color={theme.colors?.blue[400]}
```

Comment:
`colors` should never be undefined, we should figure out why the types for this got loosened.

Reply:
💯

The shared components in `@fedimint/ui` have an empty theme object at runtime. I suspect I'm missing something so the theme context is appropriately cascaded and observed